docker_builder/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "image" {
+  value       = "${local.registry}:${module.git_sha.stdout}"
```

Comment:
will it be good to output all new image `tags`?

Reply:
Yeah I'll make this change